test-requirements.txt

```diff
@@ -4,6 +4,7 @@ flake8-docstrings
 flake8-quotes
 libtmux
 lxml
+sphinx
```

Comment:
Why put it here? It's not related to tests...

Reply:
should we have a dev-requirements file too?

Reply:
Mypy does need it, and mypy could be considered a "test"....?

Reply:
@webknjaz let's iterate on where these should be. Currently test-requirements represents everything a dev would, or soon will.